docker/nautilus.sh

```diff
@@ -0,0 +1,257 @@
+    -p, --path_to_config           absolute path to configuration folder, folder should contain at the very least a conseil folder,
+                                   if using a modified schema, a postgres folder with a conseil.sql file. if not specified,
+                                   uses configuration files for conseil, postgres and tezos from the config folder in repo.
+                                   config folder name will also be used in docker container nomenclature(e.g. config folder name is
+                                   prod1, docker container name will be conseil-prod1, postgres-prod1, etc.),default config folder is
+                                   "local", it resides within config folder in repo
+                                   NOTE: docker volumes will be created here to create persistence
+    -t, --tezos                    stops and removes existing tezos container if it exists
+                                   and rebuilds and starts the tezos container
+    -v, --volume                   REMOVES postgres volume and postgres volume folder, use at own risk as this will
+                                   require all blocks to be replaced in the postgres database, this is necessary if
+                                   there has been a schema change as simply rebuilding the container will not replace the schema
+
+Examples:
+    $CMD -a -p /$HOME/production-environment-1
+                                   # build, initialize, and run docker containers
+                                   # for conseil, postgres, and tezos
+                                   # takes config files from production-environment-1 folder
+
+
+Report bugs to <swap@cryptonomic.tech>.
+EOF
+}
+
+# parse command line arguments
+SHORT_OPTS='ab:cdh:p:tv'
+LONG_OPTS='all,build-name:,conseil,database,help,path-to-config:,tezos,volume'
+ARGS=$(getopt -o $SHORT_OPTS -l $LONG_OPTS -n "$CMD" -- "$@" 2>/dev/null)
+#check getopt command failure
+(( $? != 0 )) && fatal "invalid options"
+eval set -- "${ARGS}"
+# set execution flags and/or execute functions for each option
+while true ; do
+    case "$1" in
+        -a|--all) CONSEIL=1 ; POSTGRES=1 ; TEZOS=1 ; shift ;;
+        -b|--build-name) build_name="$2" ; shift 2 ;;
+	    -c|--conseil) CONSEIL=1 ; shift ;;
+        -d|--database) POSTGRES=1 ; shift ;;
+        -h|--help) display_usage && exit 0 ; shift ;;
+        -p|--path-to-config) path_to_config="$2" ; shift 2 ;;
+        -t|--tezos) TEZOS=1 ; shift ;;
+        -v|--volume) VOLUME=1 ; shift ;;
+        --) shift ; break ;;
+    esac
+done
+
+# ensure necessary command line parameters were specified(man test to see usages, checks contents of a string)
+[[ -z "${CONSEIL}${POSTGRES}${TEZOS}${VOLUME}" ]] && display_usage \
+    && fatal "Please specify at least one container type (examples: -a,-c,-d,-t)."
+
+
+build_time=$(date "+%Y.%m.%d")
```

Comment:
This should include hour, minute and second.

Reply:
will cause more problems with no added benefit, but ok

Reply:
done